src/OpenTok/OpenTok.php

```diff
@@ -41,19 +41,29 @@ class OpenTok {
     public function __construct($apiKey, $apiSecret, $options = array())
     {
         // unpack optional arguments (merging with default values) into named variables
-        $defaults = array('apiUrl' => 'https://api.opentok.com', 'client' => null);
+        $defaults = array(
+            'apiUrl' => 'https://api.opentok.com',
+            'client' => null,
+            'default_timeout' => null // In the future we should set this to 2
```

Comment:
@dragonmantank How do you feel about calling this `timeout` instead of `default_timeout`? I think it might be a bit more user friendly. I like the approach of setting it to null so we don't break any user and then we can set it to `2` in one of the future releases if we want!

Reply:
I was just keeping it consistent with what the Ruby SDK had the parameter called, I'm fine changing it. I would just like to it be the same option for everything. I'll update the underlying Jira ticket specs